src/utilities.cpp

```diff
@@ -121,6 +121,20 @@ std::vector<int> aste::setupMesh(precice::SolverInterface &interface, const aste
     } else {
       VLOG(1) << "Mesh Setup: 4) No Quadrilaterals are found/required. Skipped";
     }
+    if (!mesh.tetrahedra.empty()) {
```

Comment:
Does `isMeshConnectivityRequired` behave the way you expect? Is it the configuration here conditional?

Reply:
Yes, currently the isMeshConnectivityRequired is a boolean, so it's all connectivity or nothing.

Reply:
My question was more: does this function behave now differently when using LCI?

Reply:
It sets up tetrahedra whenever connectivity is required. So if you have tetrahedra and use nearest-projection (or NN with a scaled-consistent mapping) they will also be configured (but unused)